packages/ns-api/files/ns.ovpnrw

```diff
@@ -176,6 +187,24 @@ def instance2number(input_string):
     numbers = re.findall(r'\d+', input_string)
     return int(numbers[0]) if numbers else None
 
+def get_static_ip_range(network, start_ip, end_ip):
+    # Convert string inputs to IP address objects
+    network = ipaddress.ip_network(network)
+    start_ip = ipaddress.ip_address(start_ip)
+    end_ip = ipaddress.ip_address(end_ip)
+    reserved_ip = network.network_address + 1
+
+    first_ip = ipaddress.ip_address(network.network_address + 1)
```

Comment:
Maybe I'm mistaken, but how is `first_ip` different from `reserved_ip`?

Reply:
You're not mistaken, this is an error.

I'll try a different approach, but I fear we are going to drop the ns_static_pool field

Reply:
Field dropped, too complicated.

- easy, pool: 192.168.0.50-192.168.0.254
- difficult, pool: 192.168.0.2-192.168.0.50
- too difficult, pool: 192.168.0.10-192.168.0.50